components/eam/cime_config/config_component.xml

```diff
@@ -47,9 +47,10 @@
     <values modifier='additive'>
       <value compset=""                >-mach $MACH</value>
       <value compset="_EAM"            >-phys default</value>
-      <value compset="_EAM%CMIP6_"      >&eam_phys_defaults; &eam_chem_defaults;</value>
+      <value compset="^((?!SSP).)*_EAM%CMIP6_" >&eam_phys_defaults; &eam_chem_defaults;</value>
```

Comment:
Changed to use this negative lookaround for exclude SSP cases from using the config options at this line. Otherwise would not be able to get rid of '-vbs' option for standard config.